pom.xml

```diff
@@ -434,6 +434,7 @@
     <version.org.kie.workbench.app>${version.org.kie}</version.org.kie.workbench.app>
 
     <version.thorntail>2.7.0.Final</version.thorntail>
```

Comment:
we can remove this too, just be sure we have not forgotten somewhere Thorntail. PIM is migrating to Quarkus so that is not forgotten, but still.

Reply:
I took it out. Might have to add it back, depends which one of use makes it in first. Not having it does not break kie-wb-distributions.